code/modules/surgery/organs/organ_external.dm

```diff
@@ -181,7 +181,7 @@
 			I.receive_damage(brute * 0.5)
 			brute -= brute * 0.5
 
-	if(status & ORGAN_BROKEN && prob(40) && brute)
+	if(status & ORGAN_BROKEN && prob(40) && brute && owner.stat)
```

Comment:
This will only be usable while `UNCONSCIOUS` or `DEAD` due to the `owner.stat` check.

Reply:
Gotcha, good catch.